packages/react-components/package.json

```diff
@@ -17,8 +17,11 @@
   "author": "koonpeng@openrobotics.org",
   "license": "Apache-2.0",
   "dependencies": {
+    "@date-io/moment": "^1.3.13",
```

Comment:
Which version was it that caused the problem? npm may install newer versions based on semver.

Reply:
Well, that's weird my npm installed 2.9.2 as the last version =O. I'll not trust npm blindly anymore. We have ^2.10.6 now, that it's the current last version. It's working correctly now after some date format config

Reply:
It's not working again, it explodes in some situations. I'll rollback to ^1.3.13. The creators of material-ui-picker also recommend using 1.x.x.
https://material-ui-pickers.dev/getting-started/installation#peer-library